### PR TITLE
Add cidr_blocks_extra parameter

### PR DESCRIPTION
**Pull request by JonRoma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ The following arguments are supported:
 
 * `port` - (Required) Server port
 * `vpc` - (Required) The VPC name
+* `cidr_blocks_extra` - (Optional) List of additional CIDR blocks granted access to the server security group. Defaults to an empty list.
```
Comment (ddriddle) [inline README.md:43]: I recommend you drop the word `extra` in the name `cidr_blocks_extra`. In the future we may want to add for example `cidr_blocks_ipv6`. The fact that the list is optional I think makes it clear enough that it is well extra.

I also recommend that you specify that it is inbound access in the description.

File: server.tf [hunk 2]
```diff
@@ -7,9 +11,19 @@ resource "aws_security_group_rule" "server-ingress" {
   security_group_id        = aws_security_group.server.id
 }
 
+resource "aws_security_group_rule" "server-ingress-extra" {
+  count             = length(var.cidr_blocks_extra) > 0 ? 1 : 0
```
Comment (ddriddle) [inline server.tf]: Please put a newline after `count` per the Terraform [style guide](https://www.terraform.io/docs/configuration/style.html).

File: variables.tf [hunk 3]
```diff
@@ -1,3 +1,9 @@
+variable "cidr_blocks_extra" {
```
Comment (ddriddle) [inline variables.tf:1]: Drop the `_extra` per the comments above.

**Review by ddriddle — CHANGES_REQUESTED** [review]
Looks good overall. Please review my comments.

**Comment by JonRoma** [comment]
@ddriddle, unfortunately we found that this does not work. The RDS instance has public_accessibility set to true, but the subnets associated with the instance are all private subnets. Would like some debug time with you Thursday if possible.

File: README.md (hunk 1 above)
Comment (JonRoma) [reply README.md:43]: Kala and I see your point, but we think the word "extra" clarifies the intent.

Regarding IPv6, I have been hearing about it for 15+ years, and yet it has never reached a state where it affects our work. All of our modules doing networking will need to be reworked if and when IPv6 becomes relevant to us, so we agreed to put this off until that day comes – and we might all be retired before that happens! :-)

**Review by cmaturi — APPROVED** [review]

**Pull request merged**